app/routers/pro_registration.py

```
@@ -0,0 +1,90 @@
+with the FastAPI application, eliminating import-side-effects and making
+PRO route registration explicit and testable.
+"""
+
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from fastapi import FastAPI
+    from fastapi.routing import APIRouter
```

Flake8 (7.3.0)

[error] 'fastapi.routing.APIRouter' imported but unused

(F401)